.golangci.yml

```diff
@@ -58,3 +60,18 @@ issues:
     - linters:
         - lll
       path: "pkg/subctl/operator/common/embeddedyamls/yamls.go"
+
+    # Allow dot imports for `fake` directory and e2e tests
+    - linters:
+        - stylecheck
+      text: "ST1001: "
+      path: _test_suite\.go
+
+    - linters:
+        - stylecheck
+      text: "ST1003: "
+      path: pkg/subctl/operator/common/embeddedyamls/yamls.go
+
+    - linters:
+        - stylecheck
+      text: "ST1003: struct field BrokerK8sApiServer"
```

Comment:
What is this rule for? It's not associated with a path.

Reply:
Its to ignore warnings suggesting changing it to `BrokerK8sAPIServer`